autotest/pymod/test_py_scripts.py

```diff
@@ -37,20 +37,29 @@
 # Return the path in which the Python script is found
 #
 
+# path relative to gdal root
+utils_subdir = 'swig/python/osgeo/utils'
+samples_subdir = 'swig/python/samples'
+samples_path = '../../gdal/' + samples_subdir
 
-def get_py_script(script_name):
 
-    for subdir in ['scripts', 'samples']:
-        try:
-            # Test subversion layout : {root_dir}/gdal, {root_dir}/autotest
-            test_path = os.path.join(os.getcwd(), '..', '..', 'gdal', 'swig', 'python', subdir)
-            test_file_path = os.path.join(test_path, script_name + '.py')
-            os.stat(test_file_path)
-            return test_path
-        except OSError:
+def get_data_path(dir):
+    return f'../{dir}/data/'
+
+
+def get_py_script(script_name):
+    # how to get to {root_dir}/gdal from ...
+    base_test_paths = [
+        # current layout (git) : {root_dir}/autotest/X
+        os.path.join(os.getcwd(), '..', '..', 'gdal'),
+        # old layout (FrankW's) : {root_dir}/gdal/autotest/X
+        os.path.join(os.getcwd(), '..', '..'),
```

Comment:
I don't think that's relevant anymore indeed

Reply:
OK. removed.